### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by corrupt**
Added a few words regarding installation and usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,26 @@ again, and was frustrutated when I couldn't find a similar plugin for vim, so
...
+### Installation
+
+Using vim >=8.0, the builtin *packages* system is recommended:
+
+```bash
+mkdir -p .vim/pack/plugins/start
+cd .vim/pack/plugins/start
+git clone https://github.com/davidbeckingsale/writegood.vim.git writegood
+```
+Writegood is now started automatically. Note that the name of the *plugins* directoy can be chosen freely.
```
Comment (davidbeckingsale) [inline README.md]: There is a typo here.

File: README.md [hunk 2]
```diff
@@ -26,6 +26,26 @@ again, and was frustrutated when I couldn't find a similar plugin for vim, so
 after reading Steve Losh's 'Learn Vimscript the Hard Way' I decided to write one
 for myself.
 
+### Installation
+
+Using vim >=8.0, the builtin *packages* system is recommended:
```
Comment (davidbeckingsale) [inline README.md]: I don't think I would recommend this. Many people use vundle, pathogen, etc. to handle this stuff in a more automatic way.

**Review by davidbeckingsale — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (corrupt) [reply README.md]: Actually vim's package mechanism works quite similar to pathogen, without the overhead. But I can easily add a few lines to that to also include pathogen, vundle, vim-plug and NeoBundle.

**Comment by corrupt** [comment]
Rewrote that section, copying fatih/vim-go. Please check if it's OK for you now.

**Pull request merged**